src/avtt/content-script.js

```diff
@@ -0,0 +1,580 @@
+    let modifier = request.modifier;
+    // Custom skill;
+    if (request.ability === "--" && request.character.abilities.length > 0) {
+        let prof = "";
+        let prof_val = "";
+        let reliableTalent = false;
+        if (request.proficiency === "Proficiency") {
+            prof = "PROF";
+            prof_val += request.character.proficiency;
+            reliableTalent = request.reliableTalent;
+        } else if (request.proficiency === "Half Proficiency") {
+            prof = "HALF-PROFICIENCY";
+            prof_val += "floor(" + request.character.proficiency + " / 2)";
+        } else if (request.proficiency === "Expertise") {
+            prof = "EXPERTISE";
+            prof_val += `${request.character.proficiency} * 2`;
+            reliableTalent = request.reliableTalent;
+        }
+        const d20 = reliableTalent ? "1d20min10" : "1d20";
+        return {
+            icon: "bunker-soldier-standing",
+            color: "#7ED321",
+            title: request.skill + " Skill Check",
+            message: template([
+                ...advantageRoll(request, "Skill Check", `!(${d20}${prof_val.length > 0 ? " + " + prof_val : ""}${custom_roll_dice.length > 0 ? " + " + custom_roll_dice : ""})`)
+            ])
+        }
+        
+    } else {
+        let d20 = request.reliableTalent ? "1d20min10" : "1d20";
+        if (request.silverTongue && (request.skill === "Deception" || request.skill === "Persuasion"))
+            d20 = "1d20min10";
+
+        
+        return {
+            icon: "bunker-soldier-standing",
+            color: "#7ED321",
+            title: request.skill + " Skill Check",
+            message: template([
+                ...advantageRoll(request, "Skill Check", `!(${d20}${modifier}${custom_roll_dice.length > 0 ? " + " + custom_roll_dice : ""})`)
+            ])
+        }
+    }
+}
+
+function rollAbility(request, custom_roll_dice = "") {
+    return {
+        icon: "coin-star",
+        color: "#4A90E2",
+        title: request.ability + " Ability Check",
+        message:  template([
+            ...advantageRoll(request, "Ability Check", `!(1d20${request.modifier}${custom_roll_dice.length > 0 ? " + " + custom_roll_dice : ""})`)
+        ])
+    }
+}
+
+function rollSavingThrow(request, custom_roll_dice = "") {
+    return {
+        icon: "report-problem-triangle",
+        color: "#F5A623",
+        title: request.ability + " Saving Throw",
+        message:  template([
+            ...advantageRoll(request, "Saving Throw", `!(1d20${request.modifier}${custom_roll_dice.length > 0 ? " + " + custom_roll_dice : ""})`)
+        ])
+    }
+}
+
+function rollInitiative(request, custom_roll_dice = "") {
+    return {
+        icon: "arrow-down-9",
+        color: "#7ED321",
+        title: "Initiative",
+        message:  template([
+            ["Initiative", `i!(1d20${request.initiative}${custom_roll_dice.length > 0 ? " + " + custom_roll_dice : ""})`]
+        ])
```

Comment:
Does astral have a "add to tracker" sort of functionality ?

Reply:
From what I read in their roll reference, it seems so by prefixing the roll with an `i`. From how we used it till now, it seems to work for the GM.